src/PowerShell/Utilities/ScriptBuilder.ts

```diff
@@ -0,0 +1,16 @@
+export default class ScriptBuilder {
```

Comment:
Should it be a separate class or a function in ServicePrincipalLogin, since it depends on the same state as login object.